lab/lab2/js/part3-application.js

```diff
@@ -31,18 +31,28 @@ var resetMap = function() {
   /* =====================
     Fill out this function definition
   ===================== */
+_.each(myMarkers,function(markers){
+  map.removeLayer(markers)
+  });
+  myMarkers=[]
 };
 
+
 /* =====================
   Define a getAndParseData function to grab our dataset through a jQuery.ajax call ($.ajax). It
   will be called as soon as the application starts. Be sure to parse your data once you've pulled
   it down!
 ===================== */
-var getAndParseData = function() {
+var phillyCrimeDataUrl = "https://raw.githubusercontent.com/CPLN692-MUSA611/datasets/master/json/philadelphia-crime-snippet.json";
+
+var getAndParseData = function(){
+  $.ajax(phillyCrimeDataUrl).done(function(ajaxResponseValue){
+    var jsdata = JSON.parse(ajaxResponseValue);
   /* =====================
     Fill out this function definition
   ===================== */
-};
+});
+
```

Comment:
It looks like your plotData() function is being defined inside of another function, which makes it unable to be called.

This may be a matter of fixing some curly brackets.

But either way, it is not able to be run. 

Were you able to plot anything for Lab2-Part3?